File: BeeCrowd/Python/1185.py
count, add = 0.0, 0.0
lim, c = 12, input()
for x in range(lim):
    for y in range(lim):
        n = float(input())
        if (y < lim-1-x):
            add += n
            count += 1

print('%.1lf' %(add if (c=='S') else add/count) )

File: BeeCrowd/Cpp/1185.cpp
#include <stdio.h>
int main() {
  double count = 0.0, add = 0.0, n;
  char c;
  int x, y, lim = 12;
  scanf("%s", &c);
  for (x = 0; x < lim; ++x) {
    for (y = 0; y < lim; ++y) {
      scanf("%lf", &n);
      if (y < lim - 1 - x) {
        add += n;
        count += 1;
      }
    }
  }

  printf("%.1lf\n", (c == 'S') ? add : add / count);

  return 0;
}
